main.cpp

```diff
@@ -0,0 +1,112 @@
+#include <string>
+#include <VSScript4.h>
+#include <elements.hpp>
+
+using namespace cycfi::elements;
+
+int main(int argc, char** argv)
+{
+    try {
+        // Init variables
+        int default_frame_number = 0;
+        const char *script_file_path{[&] {
+            return argv[1] ? argv[1] : throw std::runtime_error{"VSPreview error: Script file not specified"};
```

Comment:
the standard doesn't promise you anything about contents of argv outside the range argv[0] to argv[argc-1]. check `argc` instead